app/routes/goals.py

```diff
@@ -0,0 +1,86 @@
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, abort, request
+from app.routes.helper_functions import validate_model
+
+
+bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    
+    if not request_body:
```

Comment:
It might be a better idea to check for specific keys in the request body, so you can handle a case where a required value, e.g. "title" in this case, isn't passed in, you respond with a 400. As it is now, if I send a POST request with a body `{ "foo": "bar" }`, your application will respond with a 500.